### DISCUSSION TITLE
Silk.NET/blog/dec-2021/structure-chaining

### DISCUSSION THREAD

**Discussion by giscus** [dotnet/Silk.NET#1633] [Webpage Comments]
# Silk.NET/blog/dec-2021/structure-chaining

With the release of Silk.NET 2.11 we've enhanced the usability of Silk.NET.Vulkan by addressing one of its pain
points, specifically building the Singly Linked Lists (or chains) of
structures that are frequently used to pass information to, and from, the API.

https://dotnet.github.io/Silk.NET/blog/dec-2021/structure-chaining.html

**Comment by SupinePandora43** [comment]
Woah! That's so cool
